system/jlib/jtrace.cpp

```diff
@@ -1285,16 +1285,26 @@ std::unique_ptr<opentelemetry::sdk::trace::SpanProcessor> CTraceManager::createP
     if (!exporter)
         return nullptr;
 
-    if (exportConfig->getPropBool("batch/@enabled", false))
+    if (exportConfig->getPropBool("batch/@enabled", true))
     {
         //Groups several spans together, before sending them to an exporter.
-        //MORE: These options should be configurable from batch/@option
-        opentelemetry::v1::sdk::trace::BatchSpanProcessorOptions options; //size_t max_queue_size = 2048;
-                                                                        //The time interval between two consecutive exports
-                                                                        //std::chrono::milliseconds(5000);
-                                                                        //The maximum batch size of every export. It must be smaller or
-                                                                        //equal to max_queue_size.
-                                                                        //size_t max_export_batch_size = 512
+        opentelemetry::v1::sdk::trace::BatchSpanProcessorOptions options;
+        /**
+         * The maximum buffer/queue size. After the size is reached, spans are
+         * dropped.
+         */
+        options.max_queue_size = exportConfig->getPropInt("batch/@maxQueueSize", 2048);
+
+        /* The time interval between two consecutive exports. */
+        options.schedule_delay_millis = std::chrono::milliseconds(exportConfig->getPropInt("batch/@scheduledDelayMillis", 5000));
+
+        /**
+         * The maximum batch size of every export. It must be smaller or
+         * equal to max_queue_size.
+         */
+        options.max_export_batch_size = exportConfig->getPropInt("batch/@maxExportBatchSize", 512);
+
+        DBGLOG("Tracing exporter set to batch mode");
```

Comment:
Logging at this point has caused problems in the past, probably better not to include.